react/features/base/react/components/web/NavigateSectionListItem.js

```diff
@@ -6,17 +6,32 @@ import Container from './Container';
 import Text from './Text';
 import type { Item } from '../../Types';
 
-type Props = {
+export type Props = {
```

Comment:
Is the export necessary? I may be missing where it's being used outside of this file.